geopandas/tests/test_geodataframe.py

```diff
@@ -372,6 +368,20 @@ def test_transform(self):
         utm = lonlat.to_crs(epsg=26918)
         self.assertTrue(all(df2['geometry'].geom_almost_equals(utm['geometry'], decimal=2)))
 
+    def test_to_crs_geo_column_name(self):
+        """ Test to_crs() with different geometry column name (GH#339) """
```

Comment:
True, we've got quite a few docstrings that do that, but I agree and changed this one.